package.json

```diff
@@ -31,6 +31,6 @@
   },
   "license": "MIT",
   "dependencies": {
-    "email-addresses": "^3.1.0"
+    "email-addresses": "^4.0.0"
```

Comment:
we aren't making any incompatible changes so a minor version bump should suffice.